network/load-balancer/concepts.mdx

```diff
@@ -172,6 +184,30 @@ See [balancing-methods](#balancing-methods).
 
 <Concept>
 
+## Private Load Balancer
+
+A Load Balancer is defined as private when you choose the "private" [accessibility](#accessibility) option during Load Balancer creation. A Private Load Balancer has the following characteristics:
+
+- It has no public IP address for sending requests or initiating TCP connections.
+- It only listens to requests or connections sent to its interface(s) on the [Private Network(s)](/network/vpc/concepts/#private-networks) it is attached to. It is not accessible over the public internet.
+- Like a public Load Balancer, it can be attached to up to eight different Private Networks.
+- It can be configured or deleted using the Scaleway API, console, CLI, Terraform or other devtools.
+- It provides its metrics to [Scaleway Cockpit](/observability/cockpit/concepts/#cockpit), even though there is no traffic.
+- It does not allow the use of a Let's Encrypt [certificate](#certificate) - only imported certificates are supported.
```

Comment:
@RoRoJ I just thought of a missing limitation: private LB doesn't support multicloud IP addresses since it is not directly connected to the Internet and therefor we cannot guarantee a route to these external backend servers.

Reply:
Thanks, I added this info